rlclientlib/logger/file/file_logger.cc

```diff
@@ -0,0 +1,39 @@
+#include <sys/stat.h>
+#include "file_logger.h"
+#include "err_constants.h"
+#include "api_status.h"
+#include <fstream>
+namespace reinforcement_learning { namespace logger { namespace file {
+
+  file_logger::file_logger(const std::string& file_name, i_trace* trace)
+  : _file_name(file_name),
+  _trace(trace)
+  {}
+
+  file_logger::~file_logger()
+  {
+      _file.close();
```

Comment:
This is not needed, when the object is destroyed the file will automatically be closed.

Reply:
Removed.